.github/workflow-templates/browser-testing.properties.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Browser Testing",
+  "description": "Mocha + Chai + Karma browser testing starter workflow. To use this workflow you will need to have `chai`, `mocha`, `@jsdevtools/host-environment`, and `@jsdevtools/karma-config` installed, along with a `karma.config.js` file set up. Check out the [Karma config in `fetch-har`](https://github.com/readmeio/fetch-har/blob/main/karma.conf.js) for an example of how to set this up.",
```

Comment:
I'm curious about where this description will show up when we try and set up this workflow 🤔  I was thinking we include this in a comment block in the workflow file itself so devs will see it when writing their workflow file? I think this is good enough for now though.

Reply:
Would rather it be in a README or something but I'm not actually sure how this is surfaced. Will see after I merge this in and tweak as needed.